lib/embeds/instagram.js

```diff
@@ -18,7 +18,16 @@ export default {
       : '';
 
     const script = `<script src="${protocol}//platform.instagram.com/en_US/embeds.js"></script>
-      <script>resize()</script>`;
+      <script>
+        const interval = setInterval(function () {
+          const iframe = document.querySelector('iframe.instagram-media-rendered');
```

Comment:
@ryanscottaudio This is run in an iframe container though, so the querySelector is not run on the entire page.